### DISCUSSION TITLE
[Help me]New profile for build a super

### DISCUSSION THREAD

**Discussion by EliezerJohnson** [Takumi123x/rou#3] [General]
Is there any way to generate a profile for the m13 device (sm-m135m)?
I see that it is only available for A0 devices

**Comment by EliezerJohnson** [comment]
SM-M135M partition layout:
[Layout Partition SM-M135M.txt](https://github.com/Takumi123x/rou/files/13789486/Layout.Partition.SM-M135M.txt)
But I still don't understand how to use this information to generate a new appropriate profile. Please

**Reply by Takumi123x** [reply]
Nah there exist manual option if dont have partition profile

**Comment by Sk2244** [comment]
![Screenshot_20240122-113038.png](https://github.com/Takumi123x/rou/assets/111129071/d940f217-f5a7-459f-995f-e566bfa3ee6e)

Sir,
      Please help me sir how can I fix this problem.
Grant storage permission to termux but still get this problem.
😭😭😭😭
